x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -165,6 +168,11 @@ func TestQueryConsumerChainsValidatorHasToValidate(t *testing.T) {
 	// set `providerAddr` as an opted-in validator on "chain3"
 	pk.SetOptedIn(ctx, "chain3", providerAddr)
 
+	// set max provider consensus vals to include all validators
+	params := pk.GetParams(ctx)
+	params.MaxProviderConsensusValidators = 180
```

Comment:
Does this have to be `180` for this test or it has to be smaller?

Reply:
Changed to 3 to make it clear this can be small